src/messaging/tests/echo/echo_requester.cpp

```diff
@@ -166,6 +166,40 @@ void HandleEchoResponseReceived(chip::Messaging::ExchangeContext * ec, chip::Sys
            static_cast<double>(gEchoRespCount) * 100 / gEchoCount, payload->DataLength(), static_cast<double>(transitTime) / 1000);
 }
 
+void RunPinging()
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    // Create a seperate thread to run CHIP event loop.
+    std::thread chipThread([] { chip::DeviceLayer::PlatformMgr().RunEventLoop(); });
+
+    chipThread.detach();
```

Comment:
How about StartEventLoopTask ?

Will the Shudown call in main automatically call PlatformMgr::Shutdown to close any pending task operations?

Reply:
Acked, 

We should call "chip::DeviceLayer::PlatformMgr().Shutdown()" explicitly in ShutdownChip which shut down all sub modules implicitly.

CHIP:DL: System Layer shutdown
CHIP:DL: Inet Layer shutdown
CHIP:DL: BLE layer shutdown

Reply:
Why are we creating a std::thread instead of calling 'StartEventLoopTask'? Are they not meant to do the same thing?

Reply:
Acked, I didn't notice this API can actually spawn a thread to handle CHIP event IO. Thanks!